<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtServer</class>
 <widget class="QWidget" name="QtServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtServer</string>
  </property>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_IP_Port">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>IP: 127.0.0.12 port: 12345</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje enviado</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
